03-interpolated-defaults.tf

```diff
@@ -1,11 +1,20 @@
+locals {
+  
+  sub_env = var.environment == "ptlsbox" ? "ptl-sbox" : var.environment
+  aad_group_name = contains(["ptlsbox", "ptl"], var.environment) ? "DTS Contributors (sub:dts-sharedservices${local.sub_env})" : "DTS Contributors (sub:dts-sharedservices-${local.sub_env})"
```

Comment:
This isn't correct - if we were to deploy sbox from CFT?

Reply:
I've ran a plan against SBOX env using this branch here and all looks good - https://dev.azure.com/hmcts/CNP/_build/results?buildId=170521&view=results

Reply:
As discussed - this needs to be CFT specific

Reply:
As discussed, will resolved for now